tools/triage/filters.py

```diff
@@ -86,6 +89,7 @@ def __eq__(self, other):
 
 
 class Filters:
+    @retry.retry(exceptions=JIRAError, tries=3, delay=2)
     @staticmethod
```

Comment:
```suggestion
    @staticmethod
    @retry.retry(exceptions=JIRAError, tries=3, delay=2)
```
seems to fix the problem

Reply:
:facepalm: 
Thanks for the suggestion, I was preparing for a much bigger change after I saw the error

Reply:
I'd like changing those functions to just regular methods, but I'll guess it's a bigger change